flipperdevices/flipperzero-firmware
# [FL-2925] Furi: smaller crash routine

# What's new

- Furi: smaller crash routine

# Verification 

- Compile and upload
- Check if it's smaller 
- Check if it still crashes as intended

# Checklist (For Reviewer)
